README.md

```
@@ -1,94 +1,54 @@
-
 # ScryptTunes
 
-Welcome to ScryptTunes! This application allows you to control your Spotify playlist through Twitch chat commands. This guide will walk you through the installation and setup process, step-by-step.
+Control Spotify from Twitch chat. This guide covers install and the new OAuth setup.
 
-NOTE: This bot requires Spotify Premium. This is a Spotify API reqirement and there's nothing I can do about it
+Note: Requires Spotify Premium (Spotify API requirement).
 
-### Dev Announcements:
- - Channel points redeem is broken, working on a fix. I will disable it in the next release. Don't use it, just doesnt work at all!
+### Dev Announcements
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)